src/trytond_factories/account.py

```diff
@@ -31,11 +31,11 @@ class MinimalCA(factory.StubFactory):
     class MinimalEN(factory.StubFactory):
```

Comment:
I understand `MinimalES` should be migrated as well and `MinimalCA` should be dropped?